src/components/Profile.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
```

Comment:
- [ ]  [OPTIONAL] It's great that you created the profile component, but kindly get informed that since React 17, the `import React from 'react'` statement is not needed in simple components. Because the latest releases of React are able to automatically transform JSX without using `React.createElement`. However, you will still need react imported to use hooks and other functionalities.